indico/modules/events/management/forms.py

```diff
@@ -270,6 +268,20 @@ def _init_visibility(self, event):
             self.visibility.warning = _("This event's visibility is currently limited by that of '{}'.").format(
                 real_horizon.title)
 
+    def validate_permissions(self, field):
+        event = self.event
+        for idx, value in enumerate(field.data):
```

Comment:
for principal_fossil, permissions in field.data:

(also see the comment a few lines below to remove the need for having the index)

---

just fyi, you could do this if you needed to keep the index:

    for idx, (principal_fossil, permissions) in enumerate(field.data):